src/analysis/timeline/timelineCred.js

```diff
@@ -203,6 +158,21 @@ export class TimelineCred {
     const ftc = await _computeTimelineCred(graph, params, config);
     return new TimelineCred(graph, ftc, params, config);
   }
+
+  /**
+   * Returns the Node matching a given NodeAddressT, if such a node exists,
+   * or undefined otherwise.
+   */
+  getNodeByAddress(a: NodeAddressT): ?Node {
+    return this.graph().node(a);
+  }
+
+  /**
+   * Returns an array of cred from a given NodeAddressT from FilteredTimelineCred
+   */
+  getCredByAddress(a: NodeAddressT) {
```

Comment:
Should have an explicit type signature. (Not needed technically, but will make the code easier to read.)

Reply:
Assuming it gives interval aligned data, there should also be an API for getting the intervals.

Reply:
Also, the docstring should document what happens when the address is not present (does it throw an error? does it return undefined?)